app/views/application_submissions/show.haml

```diff
@@ -85,29 +92,32 @@ by:
       = form_tag controller: :application_submissions,
         action: :toggle_saved_for_later,
         id: @record.id do
-        %p
-          Note
+        .field.note_item
+          %p
+          <b>Note: </b>
+          %p
           = text_area_tag :note_for_later, '', required: true
-        %p
+        .text.node_item
```

Comment:
Looks like you said "node_item", did you mean "note_item"?

Reply:
Yeah sorry, it was a typo. I'll fix it.